src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -71,7 +72,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     if (userStore.isUserRegistered(username)) {
       User user = userStore.getUser(username);
-      if(password.equals(user.getPassword())) {
+      if(BCrypt.checkpw(password, user.getPassword()) == true) {
```

Comment:
Does this function not return a boolean?   I don't think the == true is necessary.